apps/studio/components/interfaces/Storage/EmptyBucketState.tsx

```diff
@@ -11,7 +11,7 @@ export const EmptyBucketState = ({ bucketType }: EmptyBucketStateProps) => {
   const config = BUCKET_TYPES[bucketType]
 
   return (
-    <aside className="mt-12 border border-dashed w-full bg-surface-100 rounded-lg px-4 py-10 flex flex-col gap-y-3 items-center text-center gap-1 text-balance">
```

Comment:
FYI @joshenlim this `mt-12` removal seems to have caused this visual bug:

<img width="2376" height="752" alt="Image" src="https://github.com/user-attachments/assets/ae52055e-ba00-40c7-b91e-c73a0d1fd430" />

Is this `mt-12` meant to be applied now perhaps to the parent `FilesBuckets/index.tsx`?

Reply:
oh i forgot to push up my fix hahaha - should be alright now 🙂